secrets.env

```diff
@@ -0,0 +1,2 @@
+PASSWORD_SALT=RFcxgy1GSXL4hApS
```

Comment:
What's the point of this file?

Reply:
Not strictly necessary, just an example that can be used with the compose file in this PR to demonstrate the feature